apps/src/componentLibrary/common/contexts/DropdownContext.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {createContext, useContext, useState, ReactNode} from 'react';
+
+const DropdownContext = createContext({
+  activeDropdownName: '',
+  /**
+   * This is a dummy function to satisfy the type checker
+   * */
+  setActiveDropdownName: (name: string) => {
+    return name as unknown as void;
```

Comment:
Typescript wants this function to be of type `(name:string) => void`, but at the same moment typescript/eslint doesn't allow us to create empty functions like `(name:string) => {}`

Reply:
Ah interesting - my preference would be to just use `eslint-disable-next-line @typescript-eslint/no-empty-function` since this is a dummy function anyway (though weirdly the TS compiler isn't complaining for me when I change it to an empty function? might just be something in my environment though).